src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_3.py

```diff
@@ -177,7 +177,7 @@
 						size_hint: (None,None)
 						width: dp(291)
 						height: dp(79)
-						on_press: root.next_screen()
+						on_press: root.next_screen(False)
```

Comment:
what difference did this make?

Reply:
This screen has an extra argument in the next_screen function which allows it to distinguish between whether it is being called by pressing the button or by the auto scheduled one which is constantly being called, and is used to stop error messages being shown if the user did not manually attempt to proceed, however the argument was being defaulted to true which meant that error messages weren't being displayed